src/Controller/XController.php

```diff
@@ -7,11 +7,11 @@
 class XController extends AbstractController
 {
     /**
-     * @Route("/")
+     * @Route("/", name="login")
```

Comment:
I wouldn't call that login yet, because the XController name  doesn't fit. Renaming the controller would be good